app/app.go

```diff
@@ -602,10 +602,18 @@ func (app *RegenApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) a
 // EndBlocker application updates every end block
 func (app *RegenApp) EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock {
 	resp := app.mm.EndBlock(ctx, req)
+
+	if ctx.BlockHeight() > 1500 { // TODO: update height
+		// we don't have EndBlocker in regen modules, so after the patch height return validatorset
+		// from the SDK module manager.
+		return resp
+	}
+
 	events, vals := app.smm.EndBlock(ctx, req)
 	if len(resp.ValidatorUpdates) > 0 && len(vals) > 0 {
 		panic("validator EndBlock updates already set by the SDK Module Manager")
 	}
+
 	resp.ValidatorUpdates = vals
 	resp.Events = append(resp.Events, events...)
```

Comment:
we should still append events from our custom `smm` and add them to the `resp.Events`.

Reply:
jk none of the modules return anything from this 🥴